src/auto/selection_data.rs

```diff
@@ -21,9 +21,11 @@ impl SelectionData {
     //    unsafe { TODO: call ffi::gtk_selection_data_get_data() }
     //}
 
-    //pub fn get_data_type(&self) -> /*Ignored*/Option<gdk::Atom> {
-    //    unsafe { TODO: call ffi::gtk_selection_data_get_data_type() }
-    //}
+    pub fn get_data_type(&self) -> Option<gdk::Atom> {
```

Comment:
@Susurrus please test any of these functions to be sure that `from_glib_none` works on atom.